package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export -o build",
     "start": "next start"
   },
   "dependencies": {
     "next": "9.5.5",
+    "next-pwa": "^3.1.5",
     "react": "17.0.0",
     "react-dom": "17.0.0",
+    "react-icons": "^3.11.0",
```

Comment:
I'd rather create `Icon` component that takes `icon` props as the icon itself + `className` to pass proper styles like size, color and so on. That package is pretty heavy, I'd be happy to go with https://www.npmjs.com/package/@mdi/font for example and extract to build only the icons we need

Reply:
so what the package is heavy?

Reply:
we may not need all of the icons, moreover `mdi` exports svg that are pretty lightweight and treeshakeable - we could take only icons we need 😉